README.md

```diff
@@ -1,2 +1,151 @@
-# web-component-rise-playlist
-Web component that rotates between different pieces of content on an HTML page
+# Rise Playlist Web Component
+
+## Introduction
+`rise-playlist` is a web component that rotates between different pieces of content on an HTML page. Each piece of content is wrapped instead of a [`rise-playlist-item` web component](https://github.com/Rise-Vision/web-component-rise-playlist-item).
+
```

Comment:
@donnapep I think this line should read: "Each piece of content is wrapped **inside**". Also, maybe remove "web component" from the link [ ] and put it after on its own. Looks a bit weird as it is right now.